tools/cask/main.go

```diff
@@ -0,0 +1,270 @@
+// Copyright 2019 Microsoft Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// The package provides a transparent pass-through
+// for the Custodian CLI to a Custodian Docker Image
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/jsonmessage"
+	"github.com/docker/docker/pkg/stdcopy"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+)
+
+const CONTAINER_HOME string = "/home/custodian/"
+const IMAGE_NAME string = "cloudcustodian/c7n:latest"
```

Comment:
can we source this from an environment variable with this value as the default. CUSTODIAN_IMAGE perhaps?

Reply:
Yeah good idea.  I was avoiding this as I didn't want to take on any cask-specific CLI right now, but just using an `ENV` is an easy compromise.